control/trajectory_follower/src/mpc.cpp

```diff
@@ -217,6 +218,38 @@ void MPC::setReferenceTrajectory(
     }
   }
 
+  /* extend terminal points */
+  {
```

Comment:
Please make this part a separate function. And the purpose of this function is unclear to other developers. Please write comments.

Reply:
fix in 8dc33ca4f5f0fa43396965924f878ddb787151b2

Reply:
Seems not addressed in https://github.com/autowarefoundation/autoware.universe/commit/8dc33ca4f5f0fa43396965924f878ddb787151b2 ?